iguana/json_reader.hpp

```diff
@@ -92,6 +95,45 @@ IGUANA_INLINE void parse_item(U &value, It &&it, It &&end) {
   parse_item(reinterpret_cast<T &>(value), it, end);
 }
 
+template <char_t U, class It>
+IGUANA_INLINE void parse_item(U &value, It &&it, It &&end) {
+  // TODO: this does not handle escaped chars
```

Comment:
Can we implement the TODO?

Reply:
No, it's not finished yet. I'll do it later